internal/datastore/crdb/crdb_test.go

```diff
@@ -251,7 +251,7 @@ func TestWatchFeatureDetection(t *testing.T) {
 		{
 			name: "rangefeeds disabled",
 			postInit: func(ctx context.Context, adminConn *pgx.Conn) {
-				_, err = adminConn.Exec(ctx, `SET CLUSTER SETTING kv.rangefeed.enabled = false;`)
+				_, err := adminConn.Exec(ctx, `SET CLUSTER SETTING kv.rangefeed.enabled = false;`)
```

Comment:
When I turned on `-race` this was triggering it 🤦‍♀️

Reply:
no, because this code is technically correct, the `postInit`s are overriding an `err` that is declared outside of it.